release_notes/44-release-notes.adoc

```diff
@@ -645,6 +646,27 @@ The following changes are included:
 
 * This release updates the process to ensure that Sensor completes the TLS handshake during secret synchronization. If this does not happen, the process can freeze and leave Sensor with incomplete data stores and a corrupted state, even if Sensor appears healthy.
 
+[id="resolved-in-version-445_{context}"]
+=== Resolved in version 4.4.5
+
+*Release date*: 29 August 2024
+
+This release of {product-title-short} fixes the following bugs:
+
+* Fixed an issue where Sensor could not identify pull secrets because the annotation was changed from `kubernetes.io service-account.name` to `openshift.io/internal-registry-auth-token.service-account` in the {ocp}.
```

Comment:
```suggestion
* Fixed an issue where Sensor could not identify pull secrets because the annotation was changed from `kubernetes.io service-account.name` to `openshift.io/internal-registry-auth-token.service-account` in {ocp}.
```